pkg/controller/networkpolicy/networkpolicy_controller_perf_test.go

```diff
@@ -187,17 +216,18 @@ func getXLargeScaleWithClusterScopedNetpol() ([]*corev1.Namespace, []*networking
 				pods = append(pods, newPod(namespace, fmt.Sprintf("pod-%d-%d", j, k), labels))
 			}
 		}
-		return namespaces, networkPolicies, pods
+		return namespaces, networkPolicies, nil, pods, nil
 	}
-	namespaces, networkPolicies, pods := getXObjects(1000, getObjects)
+	namespaces, networkPolicies, _, pods, _ := getXObjects(1000, getObjects)
 	return namespaces, networkPolicies, pods
 }
 
-func testComputeNetworkPolicy(t *testing.T, maxExecutionTime time.Duration, namespaces []*corev1.Namespace, networkPolicies []*networkingv1.NetworkPolicy, pods []*corev1.Pod) {
+func testComputeNetworkPolicy(t *testing.T, maxExecutionTime time.Duration, namespaces []*corev1.Namespace, networkPolicies []*networkingv1.NetworkPolicy, anp []*v1alpha1.NetworkPolicy, pods []*corev1.Pod, externalEntities []*v1alpha2.ExternalEntity) {
```

Comment:
this can be simplified too.
```suggestion
func testComputeNetworkPolicy(t *testing.T, maxExecutionTime time.Duration, namespaces []*corev1.Namespace, networkPolicies []runtime.Object, entities []runtime.Object) {
```

Reply:
done.